src/asmcnc/production/z_head_qc_jig/z_head_qc_pcb_set_up.py

```diff
@@ -443,6 +442,7 @@ class ZHeadPCBSetUp(Screen):
     number_of_drivers = 4
 
     x_current = str(single_stack_dual_driver_x_current)
+    y_current = str(default_y_current)
```

Comment:
why are these strings?

Reply:
I set all of the outputs that would be set by screen selection to be strings, as selected values all get passed through this function: 

`    def set_value_to_update_to(self, text_obj, radio_button):
        if radio_button != None: 
            radio_button.state ='down'
            radio_button.active = True
        value_to_set = re.findall('[0-9.]+', text_obj.text)[0]
        return value_to_set`
        
and sometimes the "value_to_set" is a string value (e.g. FW version).

In hindsight I could have wrapped the function call in `int()`, rather than setting these values to be strings, but I'm honestly not sure which is better from a readability perspective? Alternatively, it _could_ return a string only if it can't convert the return value to an int... but that seems worse?

Reply:
Yep that makes sense. 

I just questioned it as in ser conn a lot of variables are initialised or stored as strings where int/float would be expected.